puppet/ci.yaml

```diff
@@ -94,125 +94,319 @@ nsidc_jenkins::jobs:
       ENV=integration
       %{hiera('harvest_nsidc_command')}
     workspace: "%{hiera('integration_workspace')}"
```

Comment:
NOTE: the only real change with this was that I moved all the ADE harvests to the bottom, prepended them with ZZOLD_, and fixed the triggers so that the Harvest NSIDC jobs will trigger the acceptance tests (before, it was a chain of harvests, with the last one triggering acceptance tests).

I've manually updated this on CI already (mostly so that there wasn't a huge list of disabled/unused jobs clogging up the main list, now they're all at the bottom).